honeycombio/resource_board_test.go

```diff
@@ -100,6 +100,11 @@ func testAccCheckBoardExists(t *testing.T, name string) resource.TestCheckFunc {
 			return fmt.Errorf("could not find created board: %w", err)
 		}
 
+		for i := range createdBoard.Queries {
```

Comment:
All changes in this file should have happened in #100.

I'm happy to move them to their own commit if you feel that's better.